pkg/lib/ssh-setup-default-key.sh

```diff
@@ -0,0 +1,18 @@
+#! /bin/sh
+
+set -euf
+
+# Create the default SSH key if necessary.  Then we dump the
+# corresponding public key to stdout.
+
+file=$1
+type=$2
+passphrase=$3
```

Comment:
Don't pass secrets as command line arguments. They are visible to everyone on the system, and tend to get into audit/pcp/other logs. Pass it through/read it from stdin instead.

Reply:
ssh-keygen doesn't have an easy option for that, but luckily we already have code to deal with that in`pkg/lib/credentials`. So, done.

Reply:
I was just thinking of replacing `passphrase=$3` with `read passphrase`, and using `.input(passphrase)` on the JS side. But of course, reusing the existing code is even better!